src/__tests__/__snapshots__/role-helpers.js.snap

```diff
@@ -0,0 +1,166 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`logRoles logs expected roles for various dom nodes 1`] = `
```

Comment:
I think this looks really good. Would love to hear what others have to say though.